src/attr.rs

```diff
@@ -194,7 +194,8 @@ fn has_newlines_before_after_comment(comment: &str) -> (&str, &str) {
             .rev()
             .take_while(|c| c.is_whitespace())
             .filter(|&c| c == '\n')
-            .count() > 1
+            .count()
+            > 1
```

Comment:
I thought so too at first, but I came round to liking them. I think that the previous formatting felt fine because we are used to it, but it gives a misleading impression of precedence (e.g., here we are not comparing `count` with `1` we're comparing the whole chain with `1`, and it means that when scanning code, you can easily miss part of an expression, *especially* when it's short